pkg/istio-agent/sds-agent.go

```diff
@@ -133,7 +132,8 @@ const (
 
 	// The type of Elliptical Signature algorithm to use
 	// when generating private keys. Currently only ECDSA is supported.
-	eccSigAlg = "ECC_SIGNATURE_ALGORITHM"
+	eccSigAlg       = "ECC_SIGNATURE_ALGORITHM"
+	keyfactorCAName = "KeyfactorCA"
```

Comment:
We have other constant as well, GoogleCA I think, seems like it should be defined alongside there?

Reply:
we don't have any constant for GoogleCA, :(
![image](https://user-images.githubusercontent.com/49465776/83474690-c4253f80-a4b6-11ea-98c3-700c8c658208.png)